Documentation/app-operations.md

```diff
@@ -0,0 +1,225 @@
+# Fine-Grained Pod Manipulations
+To provide imperative application (application as in application container and the pod being the OS container) 
+operations (e.g. start|stop|inject) capabilities for finer-grained control over rkt containerization concepts 
+and debugging needs, this proposal introduces new stage1 entrypoints that will be used for manipulating applications inside pods.
+
+The motivation behind this change is the direction orchestration systems are taking
+and to facilitate this new direction. For more details, see [kubernetes#25899](https://github.com/yujuhong/kubernetes/blob/08dc66113399c89e31f6872f3c638695a6ec6a8d/docs/proposals/container-runtime-interface-v1.md).
+
+# API
+
+The envisioned workflow for the app-level CLI API is that After a pod has been started, 
```

Comment:
Pod will be started normally (maybe with a few additional configurations), I'm not sure I understand what you mean here